apps/cache-manager/src/app.test.tsx

```diff
@@ -81,7 +98,31 @@ describe('prerender cache manager', () => {
     mockGetComponent.mockImplementation(() => {
       throw new Error('test error')
     })
-    const response = await request(app).post('/3003/prerender').send(mockRequestBody).set('remote-name', 'testRemote')
+    const response = await request(app)
+      .post('/3003/prerender')
+      .send(mockRequestBody)
+      .set('remote-name', 'testRemote')
+      .set('Content-Language', 'en-GB')
     expect(response.statusCode).toBe(500)
   })
+  it('returns default value for content-language header', async () => {
+    await request(app)
+      .post('/3003/prerender')
+      .send(mockRequestBody)
+      .set('remote-name', 'testRemote')
+      .then(res => {
```

Comment:
can we use the await and check the return value like we do for the other tests, good to be consistent and all

Reply:
done